community-operators/couchbase-upstream/v1.0.0/couchbase-v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,298 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
The files for this operator don't need to be versioned, the courier is adding a "nest" command to do this for operator-registry. For this repo operators should look like this:
```
community-operator/couchbase/*
```